packages/interactivity-router/src/assets/test/scs.test.ts

```diff
@@ -0,0 +1,367 @@
+/**
+ * Internal dependencies
+ */
+import { shortestCommonSupersequence } from '../scs';
+
+/**
+ * Check the passed array is a contained sequence of the given
+ * supersequence.
+ * @param arr      Array.
+ * @param superseq Supersequence.
+ * @param isEqual  Optional comparator.
+ * @return True if the passed array is contained. False otherwise.
+ */
+function isSubsequence(
+	arr: unknown[],
+	superseq: unknown[],
+	isEqual = ( a: unknown, b: unknown ): boolean => a === b
+) {
+	let i = 0;
+	let j = 0;
+	while ( i < arr.length && j < superseq.length ) {
+		if ( isEqual( arr[ i ], superseq[ j ] ) ) {
+			i++;
+		}
+		j++;
+	}
+	return i === arr.length;
+}
+
+describe( 'shortestCommonSupersequence', () => {
+	describe( 'Basic functionality', () => {
+		it( 'should handle empty arrays', () => {
+			// Both arrays empty
```

Comment:
Also missing periods in this file.

```suggestion
			// Both arrays empty.
```

Reply:
Also fixed in https://github.com/woocommerce/gutenberg/pull/2/commits/519b45f10a28389ef124fceb336da50ed25ebf8e.